class.jetpack.php

```diff
@@ -2781,6 +2793,7 @@ public static function do_version_bump( $version, $old_version ) {
 		if ( ! $old_version ) { // For new sites
 			// Setting up jetpack manage
 			Jetpack::activate_manage();
+			Jetpack::activate_markdown();
```

Comment:
If you're aiming to get this to happen on all upgrades, I think you'll want to move it outside of this conditional